src/main/java/frc/robot/Ports.java

```diff
@@ -1,15 +1,24 @@
 package frc.robot;
 
+import com.ctre.phoenix.motorcontrol.can.TalonFXConfiguration;
+
 public final class Ports {
+    public static class ExampleSubsystem {
+        public static final int MAIN = 0;
+        public static final int AUX = 0;
+        public static final boolean IS_MAIN_INVERTED = false;
+        public static final boolean IS_AUX_INVERTED = false;
+        public static final boolean IS_MAIN_SENSOR_INVERTED = false;
+        public static final boolean IS_AUX_SENSOR_INVERTED = false;
+    }
```

Comment:
Why do you care about example subsystem

Reply:
This was there outside of my branch. At one point I deleted it, so then I just rewrote it.